### DISCUSSION TITLE
No mapvoting or round end after time run out on surf

### DISCUSSION THREAD

**Discussion by TommyGx** [kus/cs2-modded-server#34] [Q&A]
Hi,

I've installed everything today and the server itself runs fine it seems. 
But I've played surf today and it seems like that when the time runs out there is nor round end nor a voting poll. I'm also not sure if rtv is really working for me since the !rtv command didn't do anything.

Does anyone know why that is? do i need to configure something in addition? thanks in advance!

**Comment by kus** [comment]
@TommyGx due to the way this mod is setup, I cannot use any of the existing map vote plugins as they all use one text file, but I need support to change based on the mod.

The inbuilt CS2 map voting at the end unfortunately doesn't work, I have tried to make it show the proper map names and images (which you could in CS:GO) but sadly it doesn't work, which means we will need to use a plugin to solve this.

I have started building my own plugin to handle the map changing/voting which is in the server in it's basic form.

At the moment it chooses a random map from the maps in the current mod and changes it.

As an admin you can type !map in chat and choose a map to change to from the current maps in the current mod.

I'm working on adding in voting at the end of the map but busy with real life atm.
